helpers/homework.js

```diff
@@ -1,23 +1,14 @@
 const _ = require('lodash');
 
-function isGraded(submission) {
-	return typeof submission.grade === 'number' || !!submission.gradeComment || !_.isEmpty(submission.gradeFileIds);
-}
-
 function getGradingFileName(file) {
-	const parts = file.name.split('.');
-	const creationDate = file.createdAt.slice(0, 10);
-	const baseName = parts.slice(0, -1).join('.');
-	const fileEnding = _.last(parts);
-	return `${creationDate}_${baseName}_${file._id}.${fileEnding}`;
+	return file.name;
 }
 
 function getGradingFileDownloadPath(file) {
-	return `/files/file?download=true&file=${file._id}&name=${encodeURIComponent(getGradingFileName(file))}`;
+	return `/api/v3/file/download/${file.id}/${encodeURIComponent(file.name)}`;
```

Comment:
get**Grading**FileDownloadPath ..should we not only touch the submitted files? 
Is the naming of the function wrong?